### ISSUE TITLE
`gouv.fr`, `co.uk`, `com.sg`, other public suffixes are not being resolved as TLDs

### ISSUE THREAD

**Issue by flotwig** [peerigon/parse-domain#67]
We're running in to an issue (cypress-io/cypress#3717) caused by an update to `parse-domain` causing public suffixes to not be resolved as TLDs. `gouv.fr`, `co.uk`, `com.sg`, and other public suffixes are not being resolved as TLDs ever since `2.1.0`.

```js
const parseDomain = require('parse-domain')

console.log(
  parseDomain('dev.classea12.beta.gouv.fr', {
    privateTlds: true,
    customTlds: /^[\d\.]+$/
  })
)

```

With `parse-domain@2.0.0`, this results in:
```js
{ tld: 'gouv.fr', domain: 'beta', subdomain: 'dev.classea12' }
```

With `parse-domain@2.1.0`, this results in: 
```js
{ tld: 'fr', domain: 'gouv', subdomain: 'dev.classea12.beta' }
```

Potentially related: #64, #65

**Comment by jhnns** [comment]
Can you try out `v2.1.8`? Does it fix your issue?

**Comment by flotwig** [comment]
No, still the same issue.

**Comment by jhnns** [comment]
Ok, I can confirm the issue. It's in combination with the `privateTlds: true` flag. I need to look into it.

**Comment by jhnns** [comment]
This fix shipped with `v2.3.2` 🚀
